### ISSUE TITLE
calling getZoneHVACDehumidifierDXs in a measure fails

### ISSUE THREAD

**Issue by polidobj** [NREL/OpenStudio#3911]
Issue overview
--------------
Trying to call the getZoneHVACDehumidifierDXs on the model fails in 2.9.1.  It works in 2.8 and 2.9. 

## Current Behavior
function fails:
measure.rb failed with undefined local variable or method `getZoneHVACDehumidifierDXs' for #<NISTBIRDSNESTLCIAReport:0x0001fb699c7c58>

## Expected Behavior
Function succeeds.

## Steps to Reproduce
try running a measure with code like
model.getZoneHVACDehumidifierDXs.each do |dehumidifier|
..
end

### Environment
Windows 10 OS 2.9.1

### Context
developing a measure

**Comment by tijcolem** [comment]
@joseph-robertson Could you see if this still an issue with 3.0.0?  Close if now

**Comment by jmarrec** [comment]
No problem with an empty model use either ruby 2.2.x and openstudio2.9.1 or 2.5.x and openstudio 3.x. This is likely a problem with your measure or model. If you can share the measure I could take a quick look as a courtesy.

Tests (the prompt messages are my custom .pryrc echoing some stuff)

```
$ pry
You are running ruby 2.5.5p157: OpenStudio develop3
Loading '/home/julien/Software/Others/OS-build/Products/ruby/openstudio': 3.0.0-beta+22ca96301d
m/home/julien/Software/Others/openstudio-standards/lib/openstudio-standards/btap/btap.rb:56: 

[1] julien(main)> m = Model.new
=> #<OpenStudio::Model::Model:0x0000563d6f678358
 @__swigtype__="_p_openstudio__model__Model">
[2] julien(main)> m.getZoneHVACDehumidifierDXs
=> []
[3] julien(main)> 
```

```
(py38)julien@~$ pry
You are running ruby 2.2.10p489: OpenStudio develop
Loading 'openstudio': 2.9.1.3472e8b799

[1] julien(main)> m = Model.new
=> #<OpenStudio::Model::Model:0x0055e3b4bff688
 @__swigtype__="_p_openstudio__model__Model">
[2] julien(main)> m.getZoneHVACDehumidifierDXs
=> []
```

**Comment by polidobj** [comment]
Ok I'm dumbfounded as to why this suddenly works.  I tried making reduced testcase measure.  But I didn't encounter the problem.  I just commented out the code in the measure I'm working on, and wrote a version to get the data from the IDF. Now I tried the problematic code again and no problems there either.  I must have had a gremlin in my system somewhere. 

Also it didn't cross my mind to include the detail that this is a reporting measure.  But that's pointless now it seems.  I even double checked to make sure I'm still running 2.9.1.
